src/test/java/htsjdk/variant/variantcontext/VariantContextBuilderTest.java

```diff
@@ -21,6 +27,8 @@ public void before() {
 
         C = Allele.create("C");
         Aref = Allele.create("A", true);
+        G = Allele.create("G");
+        Tref = Allele.create("T", true);
```

Comment:
we need to just expose these as public members of allele.  it's stupid to have to redefine them in every test